flink-end-to-end-tests/flink-batch-sql-test/src/main/java/org/apache/flink/sql/tests/BatchSQLTestProgram.java

```diff
@@ -55,9 +57,10 @@ public static void main(String[] args) throws Exception {
         ParameterTool params = ParameterTool.fromArgs(args);
         String outputPath = params.getRequired("outputPath");
         String sqlStatement = params.getRequired("sqlStatement");
-
+        String shuffleMode = params.getRequired("shuffleMode");
         TableEnvironment tEnv = TableEnvironment.create(EnvironmentSettings.inBatchMode());
-
+        tEnv.getConfig()
+                .set(ExecutionOptions.BATCH_SHUFFLE_MODE, BatchShuffleMode.valueOf(shuffleMode));
```

Comment:
Let's decouple the program argument from the `BATCH_SHUFFLE_MODE` values.

Reply:
done.